ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -25,57 +26,86 @@ import {
 } from 'testing/stubs/institution-api-stub';
 
 import { AdminInstitutionEdit } from './admin-institution-edit';
+import { MembershipRequirements } from './admin-institution-options';
+
+const getCTAddress = () => screen.getByTestId('controlled-email-address');
```

Comment:
These don't match 1 for 1 the old queries, because I changed the prefix to get and query instead of the pre-existing find so as not to be confused with RTL's find functionality.